Backend/Models/Users/UserManager.cs

```diff
@@ -24,6 +24,14 @@ public static User Connect(WebSocket socket, string ip)
             return newUser;
         }
 
+        public static bool IsUserAdmin(User user) 
+        {
+            return user.Ip == "127.0.0.1";
+        }
```

Comment:
Why don't we have this function under the User class? 🤔

Reply:
validating if a user is an admin sounds like the job of the UserManager not the user? Like a person doesn't identify himself as a cop but the government identifies the person as a cop fihimta?